.github/workflows/publish-to-pypi.yml

```
@@ -0,0 +1,42 @@
+name: Publish Python 🐍 distributions 📦 to PyPI
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build-n-publish:
+    name: Build and publish Python 🐍 distributions 📦 to PyPI
+    runs-on: ubuntu-18.04
+
+    steps:
+    - uses: actions/checkout@master
+    
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.8
+    
+    - name: Install pypa/build
+      run: >-
+        python -m
+        pip install
+        build
+        --user
+    
+    - name: Build a binary wheel and a source tarball
+      run: >-
+        python -m
+        build
+        --sdist
+        --wheel
+        --outdir dist/
+        .
```

It looks like the following suggestion is the current recommendation, based on [this](https://pypa-build.readthedocs.io/en/stable/index.html)
```suggestion
    - name: Build a binary wheel and a source tarball
      run: >-
        python -m
        build
        --outdir dist/
        .
```